src/core_plugins/console/api_server/es_6_0/query.js

```diff
@@ -586,33 +586,33 @@ export default function (api) {
     // js hint gets confused here
     /* jshint -W015 */
     function_score: _.defaults({
-        __template: {
-          query: {},
-          functions: [
-            {}
-          ]
-        },
+      __template: {
         query: {},
         functions: [
-          _.defaults(
-            {
-              filter: {},
-              weight: 1.0
-            },
-            SCORING_FUNCS
-          )
-        ],
-        boost: 1.0,
-        boost_mode: { __one_of: ["multiply", "replace", "sum", "avg", "max", "min"] },
-        score_mode: { __one_of: ["multiply", "sum", "first", "avg", "max", "min"] },
-        max_boost: 10,
-        min_score: 1.0
+          {}
+        ]
       },
-      SCORING_FUNCS
+      query: {},
+      functions: [
```

Comment:
Looks like in a couple places, like here, things are being pulled out of the `__template` section. What's the reasoning behind that?